frontend/javascripts/oxalis/view/right-menu/mapping_info_view.js

```diff
@@ -234,10 +235,9 @@ class MappingInfoView extends React.Component<Props, State> {
     if (!hasSegmentation()) {
       return "No segmentation available";
     }
-
     const availableMappings =
       this.props.segmentationLayer != null && this.props.segmentationLayer.mappings != null
-        ? this.props.segmentationLayer.mappings
+        ? [...this.props.segmentationLayer.mappings]
```

Comment:
Are you doing this to avoid mutations? I'd move this before the actual `sort` invocation to keep it together. Also, you can replace it with `availableMappings.slice().sort(...)` to make it easy to inline.